### PR TITLE
Test Fix| LPS-199611 | Master

### PR DESCRIPTION
**Pull request by evanilson-santana**
https://liferay.atlassian.net/browse/LPS-199611

### CODE REVIEW

**Comment by evanilson-santana** [comment]
ci:test:sf

**Comment by evanilson-santana** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectUpgrade.testcase [hunk 1]
```diff
@@ -580,7 +582,7 @@ definition {
 			}
 		}
 
-		task ("Edit a Picklist and items") {
+		task ("Edit a Picklist and items") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectUpgrade.testcase]: You added an unnecessary whitespace at the end here

**Review by timpak — CHANGES_REQUESTED** [review]
@evanilson-santana please see my comment below. Once you address this, you can forward this PR

**Comment by evanilson-santana** [comment]
ci:forward

**Pull request closed without merging**